### ISSUE TITLE
[Question] How to get the confidence score of result

### ISSUE THREAD

**Issue by gachiemchiep** [novioleo/crnn.mxnet#14]
Hello @novioleo 

How to get the confidence score of output result ?

**Comment by novioleo** [comment]
every character was produced by softmax of probabilities,so you can compute the joint probability of one character or whole sentence.@gachiemchiep

**Comment by gachiemchiep** [comment]
@novioleo 

Thank you sir. i get it now. 
By the way. When i used your command to generate model
```bash
python train.py --name taobao_captcha --charset digit.txt --train_lst taobao_captcha.csv --batch_size 32 --seq_len 12 --num_label 6 --imgH 30 --imgW 100 --gpu --learning_rate 0.001
```

i had to change predic.py to make it work
```bash
    my_predictor = predict(images,(100,32),'model/digit',0,'./digit.txt',12,6,256,False)
```

Hope you can also update the document to clear that.

**Comment by gachiemchiep** [comment]
@novioleo 
And one more question.
Your network is significantly different from crnn's network. 
Is there any reason for that ?

**Comment by novioleo** [comment]
@gachiemchiep the first question,the demo program can execute in my docker environment.
for second,for the trade-off of accury and efficience i changed the network architecture.
you can try the crnn.py which similar to origin crnn network,but i can't promise the effect.

**Comment by gachiemchiep** [comment]
@novioleo 
Thank you for your kindness.
